examples/tutorials/translations/hindi/Part 06 - Federated Learning on MNIST using a CNN.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>Remove white space between <code>]</code> and <code>(</code> in links following markdown syntax. Also translator messed up author's github link username part should be <code><a href="https://github.com/LaRiffle" target="_blank">LaRiffle</a></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
got it!